packages/dev/core/src/Particles/gpuParticleSystem.ts

```diff
@@ -1502,6 +1535,65 @@ export class GPUParticleSystem extends BaseParticleSystem implements IDisposable
         return this._currentActiveCount;
     }
 
+    /** @internal */
+    public _update(emitterWM?: Matrix): void {
+        this._recreateUpdateEffect();
+
+        if ((<AbstractMesh>this.emitter).position) {
+            const emitterMesh = <AbstractMesh>this.emitter;
+            emitterWM = emitterMesh.getWorldMatrix();
+        } else {

```

Comment:
I added a check to exit the function early if `this.emitter` is `null`.

This is not necessary for normal use of `GPUParticleSystem` because if `this.emitter` is `null` then `isReady()` will return `false` and `_update` will not be called by `render()`. But it's now possible to call `_update` directly, so it is better to have this sanity check.